internal/core/run_clang_format.sh

```diff
@@ -6,8 +6,14 @@ else
 fi
 CorePath=$1
 
+if command -v clang-format-12 &>/dev/null; then
+    CLANG_FORMAT=clang-format-12
+else
+    CLANG_FORMAT=clang-format
```

Comment:
could we use default clang-format here? default version binary has different output here

Reply:
by default, do you mean clang-format-12?